validation/cross_validate_rust_python.py

```
@@ -54,12 +54,34 @@
+    )
+
+    R_c_rust = rp.blake_critical_radius(P_inf)
+    P_Blake_rust = classifier.blake_threshold()
+
+    print(f"  R_c = {R_c_rust*1e6:.4f} μm")
+    print(f"  P_Blake = {P_Blake_rust:.2f} Pa = {P_Blake_rust/1000:.2f} kPa")
+
+    # Assert correctness
+    assert math.isclose(R_c_rust, R_c_python, rel_tol=1e-4), "R_c mismatch between Python and Rust"
+    assert math.isclose(P_Blake_rust, P_Blake_python, rel_tol=1e-4), "P_Blake mismatch between Python and Rust"
+    print(f"  ✓ Rust values match Python calculations within tolerance")
```

Ruff (0.15.6)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)